libbeat/publisher/pipeline/controller.go

```diff
@@ -34,7 +34,8 @@ type outputController struct {
 	monitors Monitors
 	observer outputObserver
 
-	queue queue.Queue
+	queue     queue.Queue
+	workQueue workQueue
```

Comment:
This is another change I made, to simplify things. Now the output controller creates a work queue in it's constructor and the same one is used by all output workers across time.

Reply:
good change!